tools/rpki-rov.1

```diff
@@ -0,0 +1,28 @@
+.\"
+.\" This file is part of RTRlib.
+.\"
+.\" This file is subject to the terms and conditions of the MIT license.
+.\" See the file LICENSE in the top level directory for more details.
+.\"
+.\" Website: http://rtrlib.realmv6.org/
+.\"
+.TH "rpki-rov" "1"
+.SH NAME
+rpki-rov - rpki route origin validator
+.SH SYNOPSIS
+.B rpki-rov
+[\fIhost\fR] [\fIport\fR]
```

Comment:
Maybe we should make clear that "host" is the RPKI RTR cache server.

Btw, does the client connect by default to the well-known rpki-rtr TCP/323 port? If yes, we should make this clear and that the port is optional.

Reply:
How about adding "It connects to the rpki server described by \fIhost\fR and \fIport\fR and can be used …"
to the Description?
I believe that aligns well with other man pages.

Reply:
Sounds good.

Reply:
done